server/utils/personUtils.ts

```diff
@@ -0,0 +1,24 @@
+import dateUtils from './dateUtils'
+import stringUtils from './stringUtils'
```

Comment:
These should be imported from the `utils` index

Reply:
Sadly this creates a dependency cycle as all utils are exported from the index, so I'm leaving this as-is here.

Reply:
Oh yeah - this only applies outside the utils folder!